packages/atomic/src/components/atomic-result/atomic-result.tsx

```diff
@@ -1,6 +1,5 @@
 import {Component, h, Prop, Element, Listen} from '@stencil/core';
 import {Result, SearchEngine} from '@coveo/headless';
-import {bindLogDocumentOpenOnResult} from '../../utils/result-utils';
```

Comment:
Can the `bindLogDocumentOpenOnResult` function be deleted? Or is the function being used in other files?